bigtable/autoscaler/strategies.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2017 Google Inc.
```

Comment:
Ya doc is still a WIP so once we have a draft of that I'll take another look at this and if it's pointlessly complicating things I'll move it back into `autoscaler.py`.